test/integration/homepage_popular_links_test.rb

```diff
@@ -135,6 +136,20 @@ class HomepagePopularLinksTest < JavascriptIntegrationTest
     end
   end
 
+  context "#publish" do
+    setup do
+      Services.publishing_api.stubs(:publish)
```

Comment:
Do we need this stubbing? Commenting it out doesn't seem to cause any ill-effect.